website/docs/d/avs_private_cloud.html.markdown

```diff
@@ -0,0 +1,96 @@
+---
+subcategory: "Avs"
+layout: "azurerm"
+page_title: "Azure Resource Manager: Data Source: azurerm_avs_private_cloud"
+description: |-
+  Gets information about an existing Azure Vmware Solution Private Cloud.
+---
+
+# Data Source: azurerm_avs_private_cloud
+
+Use this data source to access information about an existing Azure Vmware Solution Private Cloud.
+
+## Example Usage
+
+```hcl
+data "azurerm_avs_private_cloud" "example" {
+  name                = "existing-avs-private-cloud"
+  resource_group_name = "existing-resgroup"
+}
+
+output "id" {
+  value = data.azurerm_avs_private_cloud.example.id
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of this Azure Vmware Solution Private Cloud.
+
+* `resource_group_name` - (Required) The name of the Resource Group where the Azure Vmware Solution Private Cloud exists.
+
+## Attributes Reference
+
+In addition to the Arguments listed above - the following Attributes are exported: 
+
+* `id` - The ID of the Azure Vmware Solution Private Cloud.
+
+* `location` - The Azure Region where the Azure Vmware Solution Private Cloud exists.
```

Comment:
all of these want to be `VMware Private Cloud` rather than `Azure Vmware Solution Private Cloud`